src/config.rs

```diff
@@ -388,14 +393,18 @@ pesto
 
     #[test]
     fn test_command_without_whitespace() -> std::io::Result<()> {
-        let contents = "0
+
+        let contents = "
```

Comment:
The test is AFAIK correct. The error is not in the `real_line_no` variable as that's the **keysym** line. Since the error is in the command, it should return the line of the invalid command. Since commands are always one line after the keysym, it should be like this: 

```rust
return Err(Error::InvalidConfig(ParseError::CommandWithoutWhitespace(real_line_no + 1)));
```

Reply:
Sorry forgot about the line number :P